project/src/main/webapp/index.html

```diff
@@ -1,3 +1,95 @@
-<script>
-	window.location.replace("/pages/courses.html");
-</script>
\ No newline at end of file
+<!DOCTYPE html>
```

Comment:
Good start on this, let's make sure to go through and iron out all the wording and check for any spelling errors. We want our master branch to be demo ready at all times.